### PR TITLE
A range of small fixes + Hallowing UF2

### PR DESCRIPTION
**Pull request by ladyada**

### CODE REVIEW

**Comment by ladyada** [comment]
@dhalbert wanna take a look in the next few days?

File: boards/feather_m0_express/board_config.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#ifndef BOARD_CONFIG_H
+#define BOARD_CONFIG_H
+
+#define VENDOR_NAME "Adafruit Industries"
+#define PRODUCT_NAME "Feather M0"
```
Comment (dhalbert) [inline boards/feather_m0_express/board_config.h]: call this "Feather M0 Express" to distinguish it from plain "Feather M0" ?

File: boards/hallowing_m0/board_config.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#ifndef BOARD_CONFIG_H
+#define BOARD_CONFIG_H
+
+#define VENDOR_NAME "Adafruit Industries"
+#define PRODUCT_NAME "Hallowing M0"
+#define VOLUME_LABEL "HALLOWBOOT"
+#define INDEX_URL "http://adafru.it/3900"
+#define BOARD_ID "SAMD21G18A-Feather-v0"
```
Comment (dhalbert) [inline boards/hallowing_m0/board_config.h]: "Feather" -> "Hallowing"?

File: boards/feather_m0_express/board_config.h [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+#define VOLUME_LABEL "FEATHERBOOT"
+#define INDEX_URL "http://adafru.it/3403"
+#define BOARD_ID "SAMD21G18A-Feather-v0"
```
Comment (dhalbert) [inline boards/feather_m0_express/board_config.h]: "SAMD21G18A-Feather-v0" or something like "SAMD21G18A-Feather-M0-Express-v0"?

File: boards/feather_m0_express/board_config.h (hunk 1 above)
Comment (ladyada) [reply boards/feather_m0_express/board_config.h]: ok

File: boards/hallowing_m0/board_config.h (hunk 2 above)
Comment (ladyada) [reply boards/hallowing_m0/board_config.h]: ok

File: boards/feather_m0_express/board_config.h (hunk 3 above)
Comment (ladyada) [reply boards/feather_m0_express/board_config.h]: ok

**Pull request merged**